stackdriver.go

```diff
@@ -143,10 +143,17 @@ type Options struct {
 	// Optional, but encouraged.
 	MonitoredResource monitoredresource.Interface
 
-	// MetricPrefix overrides the prefix of a Stackdriver metric type names.
-	// Optional. If unset defaults to "OpenCensus".
+	// MetricPrefix overrides the prefix of a Stackdriver metric display names.
```

Comment:
This documentation was incorrect before. The MetricPrefix only affect the display name.